templates/forum/reply.html

```diff
@@ -12,7 +12,17 @@
 
 <form method="post" action="." class="disable-on-submit">{% csrf_token %}
 {% include 'forum/html_explanation.html' %}
-{{form.as_p}}
+<div>
+    {{ form.non_field_errors }}
+    {{ form.body.errors }}
+<p>
+    <label for="{{ form.body.id_for_label }}">Body:</label><br>
```

Comment:
What do you think about calling this "post" or "message" instead of body?

Reply:
Done in https://github.com/MTG/freesound/pull/1318/commits/59ff0d2edd83899af58c20b50b9ab4250c5bbb0c